tests/toranj/cli/cli.py

```diff
@@ -160,6 +167,7 @@ def cli(self, *args):
         if index == 0:
             result = [
                 line for line in self._cli_process.match.group(1).decode().splitlines()
+                if not '~ Discovery Request from' in line  # TODO Remove when PR is merged.
```

Comment:
Is this still a TODO?

Reply:
I removed it. Thanks. With PR #8579 merged already we should be good.